Tests/CommandsTests/PackageToolTests.swift

```diff
@@ -1232,6 +1232,17 @@ final class PackageToolTests: CommandsTestCase {
                 public func Foo() { }
                 """
             }
+            try localFileSystem.writeFileContents(packageDir.appending(components: "Sources", "MyLibrary", "test.docc")) {
+                $0 <<< """
```

Comment:
nit: you can pass string directly no need in byte stream

Reply:
Thanks — I did try that in the previous PR where you mentioned it, but it would require manual creation of intermediate directories, so this seems better.  What's I'd like to do as follow-up is add an optional parameter about whether to create intermediate directories to the ToolSupportCore methods, and then switch over to use them.  The trade-off would be to use a string here but to explicitly have to create all directories.